CONTRIBUTING.md

```diff
@@ -177,18 +177,30 @@ a new application. The name of the application should consist of the
 component name in lowercase, with words separated by dashes (`-`). The
 deployment server will provide a remote repository URL that will
 contain the app. Add it to the list of remotes by running `git
-remote add gallery-test [deployment server git URL]`.
+remote add [component name]-test [deployment server git URL]`.
 
-#### Step 2: Edit and commit the `Procfile`
+#### Step 2: Edit and commit app-specific files
+
+##### Step 2a: Edit the `Procfile`
 Edit the `Procfile` in the repository to say `gunicorn
-app_name:server`, where `app_name` is the name of the app you want to
-deploy in the `tests/dashbio_demos/` folder. Then, commit the change
-you made to the `Procfile`.
+tests.dashbio_demos.app_name:server`, where `app_name` is the name of
+the app you want to deploy in the `tests/dashbio_demos/` folder. 
+
+##### Step 2b: Edit `config.py`
+Edit the `config.py` file in the repository such that the variable
+`DASH_APP_NAME` is set to `dash-component-name`, where
+`component-name` is the component name in lowercase, with words
+separated by dashes (`-`).
+
+##### Step 2c: Commit the changes
+Commit the `Procfile` and `config.py`, but *do not push to the
+`dash-bio` repo*!
```

Comment:
```suggestion
default remote*!
```

Reply:
@mkcor Not sure about this, since people can set other things as their default remote (e.g., the gallery). Is there a reason you would prefer not to use `dash-bio repo`?